client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -120,7 +120,7 @@ class RegistrantExtraInfoForm extends React.PureComponent {
 
 	render() {
 		const translate = this.props.translate;
-		const registrantType = this.props.contactDetails.extra.registrantType;
+		const registrantType = get( this.props, 'contactDetails.extra.registrantType', 'individual' );
```

Comment:
Do we need this? We init a value there always in the constructor so this provides defense only against a bug somewhere in executing logic giving us an empty value--somehow.

Blowing up might be better than falling to `individual`. What do you think?